doc/content/msr/msfr/thermo_model.md

```diff
@@ -0,0 +1,118 @@
+# MSFR Griffin-Pronghorn-Thermochimica Model
```

Comment:
```suggestion
# MSFR Spatially-resolved thermo-chemistry model using Thermochimica
```
I would change this because you are not running Griffin nor pgh

Reply:
Accepted into push.